lib/core/CStaticThreadPool.cc

```diff
@@ -40,9 +41,14 @@ CStaticThreadPool::~CStaticThreadPool() {
     this->shutdown();
 }
 
+void CStaticThreadPool::numberThreadsInUse(std::size_t threads) {
+    threads = std::max(std::min(threads, m_Pool.size()), 1UL);
```

Comment:
```suggestion
    threads = std::max(std::min(threads, m_Pool.size()), std::size_t{1});
```

I think `size_t` is `unsigned long long` on some platforms, so the `UL` suffix might not be portable.

Reply:
See [f047acd](https://github.com/elastic/ml-cpp/pull/2232/commits/f047acdece21675beec11357850a4b73d3defdd6).